ribs.py

```diff
@@ -0,0 +1,132 @@
+#
+# The offical LiTHe kod code skeleton for PyGame development.
+# 2020 by Edvard Thörnros
+#
+# The aim of this file is to give a collection of usable functions
+# and structures to be used as a jumping off point. Feel free
+# to change and distribute!
+
+import pygame as pg
+# For smaller cheat sheet see:
+#   https://www.lithekod.se/pygame.html
+# For official pygame documentation see:
+#   https://www.pygame.org/docs/
+
+#
+# Utility
+#
+
+running = True
+def is_running():
+    """Is the game currently running?"""
+    return running
+
+
+def quit_game():
+    """Pull the brakes and stop the game!"""
+    global running
+    running = False
+
+
+#
+# Input handling
+#
+
+def event_is(event, kind):
+    return pg.event.event_name(event.type) == kind
+
+
+current_frame_held_buttons = set()
+last_frame_held_buttons = None
+def process_events():
+    """Tells the game what buttons are pressed."""
+    global last_frame_held_buttons
+    last_frame_held_buttons = current_frame_held_buttons.copy()
+    for event in pg.event.get():
+        if event_is(event, "Quit"):
+            quit_game()
+        elif event_is(event, "KeyDown"):
+            current_frame_held_buttons.add(event.key)
+        elif event_is(event, "KeyUp"):
+            current_frame_held_buttons.remove(event.key)
+        elif event_is(event, "MouseButtonUp"):
+            ...
+        elif event_is(event, "MouseButtonDown"):
+            ...
+        elif event_is(event, "MouseMotion"):
+            ...
+
+
+def to_keycode(key):
+    """Takes a keycode or a character and converts it to a keycode"""
```

Comment:
Example usage would be nice (What a keycode and a character might not be obvious)